Dockerfile.deploy

```diff
@@ -0,0 +1,76 @@
+FROM alpine as artifacts
+
+COPY . /usr/src/
```

Comment:
Do you really need to copy sources as the first step of the building? I mean we should order our steps from most stable to most changeable. After any changes in the current directory you will install zip, curl, and something else. Can we use here the same practices as we use in HOPER? Example:
1. Absolutely stable steps at the beginning https://github.com/latera/hoper/blob/master/Dockerfile.base#L4-L95
2. Not so stable steps in the middle https://github.com/latera/hoper/blob/master/Dockerfile.base#L97-L115
3. Most changeable steps at the end https://github.com/latera/hoper/blob/master/Dockerfile.base#L117-L142

Reply:
Do you really want to copy everything from current dir? Dockerfiles, logs, temporary directories?